modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
I'm really against having yet-another-flag. Properly layout the bindings folder to match a gem-like folder structure (with bin/ lib/ and test/) and do `rock_add_ruby_package(lib/my_package_name)`

Example: https://github.com/rock-core/drivers-transformer/blob/master/ruby/CMakeLists.txt